modules/otel-config-multicluster.adoc

```diff
@@ -0,0 +1,162 @@
++
+.Example ClusterRole
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  <1>
+  <2>
+- apiGroups: ["", "config.openshift.io"]
+  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+----
+<1> The `k8sattributesprocessor` requires permissions for pods and namespace resources.
+<2> The `resourcedetectionprocessor` requires permissions for infrastructures and status.
+
+. Bind the cluster role to the service account.
++
+.Example ClusterRoleBinding
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-deployment
+  namespace: otel-collector-<example>
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+----
+
+. Create the YAML file to define the `OpenTelemetryCollector` custom resource (CR) in the edge clusters.
++
+.OpenTelemetryCollector for the edge clusters
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: otel-collector-<example>
+spec:
+  mode: daemonset
+  serviceAccount: otel-collector-deployment
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      opencensus:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      zipkin:
+    processors:
+      batch:
+      k8sattributes:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 50
+        spike_limit_percentage: 30
+      resourcedetection:
+        detectors: [openshift]
+    exporters:
+      otlphttp:
+        endpoint: https://observability-cluster.com:443 <1>
+        insecure: false
+        compression: on
+        tls:
+          cert_file: "/path/to/server-cert.pem"
+          key_file: "/path/to/server-key.pem"
+          client_ca_file: "/path/to/client-ca.pem"
+
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger, opencensus, otlp, zipkin]
+          processors: [memory_limiter, k8sattributes, resourcedetection, batch]
+          exporters: [otlp]
+----
+<1> The Collector exporter is configured to export OTLP HTTP and points to the OpenTelemetry Collector from the central cluster.
+
+. Create the YAML file to define the `OpenTelemetryCollector` custom resource (CR) in the central cluster.
++
+.OpenTelemetryCollector for the central cluster
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otlp-receiver
+  namespace: observability
+spec:
+  mode: "deployment"
+  ingress:
+    type: route
+    hostname: "observability-cluster.com"
+    annotations:
+      acme.cert-manager.io/http01-ingress-class: openshift-default
```

Comment:
Currently cert-manager doesn't support injecting certs in route. Slack thread: https://redhat-internal.slack.com/archives/C04JVEQ8C79/p1692271285932899 https://issues.redhat.com/browse/CM-16 We need to specify the certs in receiver: and use passthrough route. https://github.com/open-telemetry/opentelemetry-operator/blob/main/tests/e2e-openshift/multi-cluster/02-otlp-receiver.yaml cc @pavolloffay @iblancasa

Reply:
I would remove the cert manager annotation from here. We could reference cert manager as a solution to provision certs if there is cert-manager product. 

I would leave cert provisioning to the end user for  now

Reply:
Suggestion to remove lines 140-141 is now applied.